### ISSUE TITLE
Add Abacus to F-Droid store

### ISSUE THREAD

**Issue by Trigant** [victorbalssa/abacus#128]
**Is your feature request related to a problem? Please describe.**
For users of degoogled androids without play store it would be awesome to find this app in the open source store f-droid

**Describe alternatives you've considered**
Add the packaged apk to the release page for manual download.

**Comment by victorbalssa** [comment]
Universal APK available here: https://github.com/victorbalssa/abacus/releases/tag/0.9.1

**Comment by AlexKalopsia** [comment]
I agree it would be nice to have the releases on F-Droid so users can also easily update to newer versions

**Comment by Trigant** [comment]
Missing universal apk in the release version 0.9.2
And still no F-Droid release. 
Pretty disappointed by now

**Comment by mss-cyclist** [comment]
I agree, a F-Droid version will make life easier for self hosters

**Comment by victorbalssa** [comment]
Maintaining a Self-Hosted F-Droid or submitting a React Native app to the official F-Droid repo can be _very_ time-consuming and complex in the long run.

I will attach a new `universal.apk` for each minor versions, patch versions will always be available via Expo EAS Updates.

However, If someone wants to submit a request to the official F-Droid repo or include Abacus to a Self-Hosted F-Droid server, I will be happy to help for the setup.

**Comment by Trigant** [comment]
How about an in app updater like the revanced manager or aliucord

**Comment by victorbalssa** [comment]
New .apk version attach to https://github.com/victorbalssa/abacus/releases/tag/0.10.2

**Comment by TickDracy** [comment]
If you don't have Play Store installed, why don't you use [Aurora Store](https://auroraoss.com/) to install Abacus?
It works pretty well for Degoogled scenarios.

Another alternative is to use [Obtainium](https://github.com/ImranR98/Obtainium) and then add this repo into it so that it can alert you whenever there is an update.
